app/views/bulk_actions/index.html.erb

```diff
@@ -1,7 +1,14 @@
 <div class='container'>
 
   <h1>Bulk Actions</h1>
-  <%= link_to 'New Bulk Action', new_bulk_action_path, class: 'btn btn-success' %>
+  <div>
+    <div style="float: left">
+      <%= link_to 'New Bulk Action', new_bulk_action_path, class: 'btn btn-success' %>
+    </div>
+    <div style="float: right">
+      <%= link_to 'Back to search', @last_search_url, class: 'btn btn-primary' %>
```

Comment:
why is this a primary action?  Seems like it should be a `btn-default` (`btn-secondary` in bootstrap 4)

Reply:
Sure, was just going for a different color, but happy to change it.